### PR TITLE
feat(tangle-dapp): Implement LS pool modal: Update commission

### LINKED ISSUE #2589: [FEAT] Implement LS pool modal: Update commission
Implement a liquid staking pool management modal that allows users with the corresponding role (root) to update the commission charged by the pool.

### LINKED ISSUE #2598: [BUG] Addresses on claim dropdown are not consistent
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

image

### Expected Behavior

- [x] The addresses should not be all capitalized in the selection and they should all be formatted with Tangle's SS58 encoding.

- [ ] The selection would be best if it also displayed the name from the extension too as it shows in the dropdown. Consistency is lost on this component let's fix this.

### LINKED ISSUE #2601: [TASK] Small UI font visibility & improvements
- [x] Increase font size of Current Era & Session on the Nomination page
- [x] Add commas to current era & session numbers on the nomination page
- [x] Make the background/overlay of the modals darker
- [x] Increase font size of the balances table on the Account page
- [x] On the Claim Airdrop page, the active network's SS58 address prefix is not being considered within the inputs, on some places
- [x] Change banner text to: Mint Liquid Staking Tokens (LSTs) to earn & restake on Tangle Mainnet.

### PR DESCRIPTION
**Pull request by yurixander**
## Summary of changes

_Provide a detailed description of proposed changes._

- ➕ Added the liquid staking pool management modal that allows setting & updating the commission rate of a specific liquid staking pool.
- 🔧 Refactored some of the text used in the Claim Airdrop page to make it more natural sounding and explicit.
- 🐛 Fixed bugs on the Claim Airdrop page related to the claiming input's address being wrongly capitalized (#2598).
- 🖌️ Updated & improved some font sizes and designs across the dApp for improved legibility (#2601).

### Proposed area of change

_Put an `x` in the boxes that apply._

- [x] `apps/tangle-dapp`
- [ ] `apps/tangle-cloud`
- [ ] `libs/webb-ui-components`

### Associated issue(s)

_Specify any issues that can be closed from these changes (e.g. `Closes #233`)._

- Closes #2589
- Closes #2598
- Closes #2601

### Screen Recording

_If possible provide screenshots and/or a screen recording of proposed change._

https://github.com/user-attachments/assets/5b3dd456-24ca-4a39-b25d-9b9d771fef03

### CODE REVIEW

**Comment by drewstone** [comment]
Don't forget to provide short video demos :)!

**Comment by yurixander** [comment]
> Don't forget to provide short video demos :)!

@drewstone Yes, on some I omit them if they're more 'technical' changes, but on new UI features like this I'll be sure to include them.

File: apps/tangle-dapp/components/LiquidStaking/LsMyPoolsTable.tsx [hunk 1]
```diff
@@ -229,6 +247,26 @@ const LsMyPoolsTable: FC<LsMyPoolsTableProps> = ({ pools, isShown }) => {
...
+    }
+
+    const selectedPool = pools.find((pool) => pool.id === selectedPoolId);
+
+    return selectedPool === undefined
+      ? null
+      : (selectedPool.commissionFractional ?? null);
+  }, [pools, selectedPoolId]);
+
+  // Reset the selected pool's ID after all the management
```
Comment (drewstone) [inline apps/tangle-dapp/components/LiquidStaking/LsMyPoolsTable.tsx:262]: Does this comment need to be split on two lines?

**Review by drewstone — APPROVED** [review]
Looks good, comments on style/comments and other misc things.

File: apps/tangle-dapp/data/liquidStaking/tangle/useLsSetCommissionTx.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+        return api.tx.lst.setCommission(poolId, [
+          commissionPerbill,
+          payeeSubstrateAddress,
+        ]);
+      },
+      [],
+    );
+
+  // TODO: Add EVM support once precompile(s) for the `lst` pallet are implemented on Tangle.
```
[the remark this answers is not in the document]
Comment (yurixander) [reply apps/tangle-dapp/data/liquidStaking/tangle/useLsSetCommissionTx.ts]: Added for all except the `setCommission` tx, checking the LST precompile, it seems to be missing that function CC @1xstj @drewstone

**Pull request merged**